### ISSUE TITLE
关于论文《TFDet: Target-aware fusion for RGB-T pedestrian detection》代码实现细节的请教

### ISSUE THREAD

**Issue by zhangchaokai335** [XueZ-phd/TFDet#10]
张学长您好，
非常感谢您在 GitHub 分享了《TFDet: Target-aware fusion for RGB-T pedestrian detection》的源代码，这对我的研究启发很大。我已经在本地尝试运行了该代码，目前能够顺利执行 。

但在深入复现的过程中，我发现仓库中似乎缺少了 有关在FLIR和M3FD数据集上的代码。由于这部分对于完整复现论文实验结果至关重要，我想请问您是否方便共享这部分代码？

我保证该代码仅用于学术研究，并在后续论文中严格引用您的工作。

再次感谢您的无私分享！

祝好

**Comment by XueZ-phd** [comment]
您好，感谢关注我们的工作，m3fd、flir的数据做好数据集即可，请问你说的代码是什么意思发自我的 iPhone在 2026年4月14日，11:27，zhangchaokai335 ***@***.***> 写道：﻿zhangchaokai335 created an issue (XueZ-phd/TFDet#10)
张学长您好，
我是来自东北大学的本科生。非常感谢您在 GitHub 分享了《TFDet: Target-aware fusion for RGB-T pedestrian detection》的源代码，这对我的研究启发很大。我已经在本地尝试运行了该代码，目前能够顺利执行 。
但在深入复现的过程中，我发现仓库中似乎缺少了 有关在FLIR和M3FD数据集上的代码。由于这部分对于完整复现论文实验结果至关重要，我想请问您是否方便共享这部分代码？
我保证该代码仅用于学术研究，并在后续论文中严格引用您的工作。
再次感谢您的无私分享！
祝好

**Comment by zhangchaokai335** [comment]
我是看到您在论文中有提到For the FLIR dataset, we use Faster R-CNN [20] as the
baseline detector and AP as the evaluation metric. We use
Swin-Transformer-Tiny [70] as the backbone network and
AdamW as the optimizer. We set the initial learning rate
to 10-4, and use random horizontal flipping and random
cropping as data augmentation techniques, following the
implementations in previous state-of-the-art work [31].
For the M3FD dataset, we use YOLOv5-Small [23] as
the baseline detector and AP as the evaluation metric.
We train YOLOv5-Small for 36 epochs. We reimplement
all comparative approaches using the same data split for a
fair comparison. For image fusion-based approaches, we first
generate fused images based on their codes, then train
YOLOv5-Small using the same settings as our method.我想看一下您当时的配置代码以及权重文件

**Comment by XueZ-phd** [comment]
这两个数据集可以查看主页中rethinking…的仓库在 2026年4月14日，15:05，zhangchaokai335 ***@***.***> 写道：﻿zhangchaokai335 left a comment (XueZ-phd/TFDet#10)
我是看到您在论文中有提到For the FLIR dataset, we use Faster R-CNN [20] as the
baseline detector and AP as the evaluation metric. We use
Swin-Transformer-Tiny [70] as the backbone network and
AdamW as the optimizer. We set the initial learning rate
to 10-4, and use random horizontal flipping and random
cropping as data augmentation techniques, following the
implementations in previous state-of-the-art work [31].
For the M3FD dataset, we use YOLOv5-Small [23] as
the baseline detector and AP as the evaluation metric.
We train YOLOv5-Small for 36 epochs. We reimplement
all comparative approaches using the same data split for a
fair comparison. For image fusion-based approaches, we first
generate fused images based on their codes, then train
YOLOv5-Small using the same settings as our method.我想看一下您当时的配置代码以及权重文件

**Comment by zhangchaokai335** [comment]
我并没有找到您说的主页中rethinking…的仓库，请你能给我完整的名字吗？我想看一下您的论文《TFDet: Target-aware fusion for RGB-T pedestrian detection》在数据集m3fd、flir上的配置代码以及最后的权重文件

**Comment by zhangchaokai335** [comment]
你好，您有看到吗？

**Comment by XueZ-phd** [comment]
https://github.com/XueZ-phd/Efficient-RGB-T-Early-Fusion-Detection#dataset

**Comment by XueZ-phd** [comment]
hi, i have response this issue #11
